cloud/ec2_test.go

```diff
@@ -389,9 +377,7 @@ func (s *EC2Suite) TestSpawnHostClassicOnDemand() {
 }
 
 func (s *EC2Suite) TestSpawnHostVPCOnDemand() {
-	pkgCachingPriceFetcher.ec2Prices = map[odInfo]float64{
-		odInfo{"Linux", "instanceType", "US East (N. Virginia)"}: .1,
-	}
+	// kim: TODO: check this test
```

Comment:
What are these todos for? are they meant for a different ticket? If yes, do we want to add the ticket number?